t/14-isotovideo.t

```diff
@@ -7,8 +7,17 @@ use Test::Warnings ':report_warnings';
 use Test::MockModule;
 use FindBin '$Bin';
 use lib "$Bin/../external/os-autoinst-common/lib";
-use OpenQA::Test::TimeLimit '20';
 use autodie ':all';
+
+# Each full isotovideo invocation (not using _exit_after_schedule=1)
+# is expected to take maximum ~6.67 seconds per run.
+#
+# There are 3 invocations, that approxs to ~20 seconds.
```

Comment:
I don't think such specific comments are a good idea. Comments lie sooner or later. Someone will add another invocation and not change this number.